src/plugins/js-eval/processTopLevelAwait.js

```diff
@@ -65,20 +63,34 @@ function processTopLevelAwait(src) {
   }
 
   // replace last node with a returnStatement of this node
-  root.program.body[root.program.body.length - 1] = b.returnStatement(last);
-
-  const iiafe = b.callExpression(
-    b.arrowFunctionExpression(
-      [],
-      b.blockStatement(root.program.body),
-      true
-    ),
-    []
-  );
-
-  iiafe.callee.async = true;
+  root.program.body[root.program.body.length - 1] = {
+    type: 'ReturnStatement',
+    argument: last
+  };
```

Comment:
Wouldn't it need to be something like this?

```js
const last = root.program.body[root.program.body.length - 1];
if (last && last.type === 'ExpressionStatement') {
  root.program.body[root.program.body.length - 1] = {
    type: 'ReturnStatement',
    argument: last.expression,
  };
}
```

https://astexplorer.net/#/gist/66ba4db7a730e060edc8423c5525fe92/b2e53e04fbae4a49e7788e105ad299cbb101783b